gc/base/GCExtensionsBase.hpp

```diff
@@ -1105,31 +1105,71 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 
 	MMINLINE MM_Heap* getHeap() { return heap; }
 
-#if defined(OMR_GC_MODRON_SCAVENGER)
 	MMINLINE void
 	setGuaranteedNurseryRange(void* start, void* end)
 	{
+#if defined(OMR_GC_MODRON_SCAVENGER)
 		_guaranteedNurseryStart = start;
 		_guaranteedNurseryEnd = end;
+#endif /* OMR_GC_MODRON_SCAVENGER */
 	}
 
 	MMINLINE void
 	getGuaranteedNurseryRange(void** start, void** end)
 	{
+#if defined(OMR_GC_MODRON_SCAVENGER)
 		*start = _guaranteedNurseryStart;
 		*end = _guaranteedNurseryEnd;
+#elif /* OMR_GC_MODRON_SCAVENGER */
+		*start = NULL;
+		*end = NULL;
+#endif /* OMR_GC_MODRON_SCAVENGER */
+	}
+
+	MMINLINE bool isRememberedSetInOverflowState()
```

Comment:
Now that these 3 RS methods are exposed outside of Scavenger compile fla, it would nice that method names are more explicit what kind of RS we are talking (since there is more then just Scavenger RS in our code base).
So, I recommend we expand these names to:
isScavengerRememberedSetInOverflowState
and similarly for 2 others related.
Since this is being used by dependent projects, you'll have keep the exisiting APIs, till depended projects get updated.

Reply:
new functions have added